yarn.lock

```diff
@@ -3313,32 +3318,7 @@ __metadata:
   peerDependenciesMeta:
     "@storybook/mdx2-csf":
       optional: true
-  checksum: ccaeaad355009eb03a911b01bba721b4ecb99ee9021238488bb6d413df02a8fe06663bc51ac80cf6b52647d0576f7841274d6ce53d1a257afc2c7b727014a12b
-  languageName: node
-  linkType: hard
-
-"@storybook/csf-tools@npm:^6.4.3":
```

Comment:
Note for others.  We needed to `yarn up @storybook/csf-tools` here, in order to get an updated version in the lockfile.  It's possible this might bite others as well who are upgrading from a previous installation of storybook 6.4 and vite builder.  It's something we should probably keep an ear out for, at least.  Relying on these kinds of sub-dependencies of storybook packages is a bit fraught.  In this case, we'd be better off not having an explicit `dependencies` list in our package.json, probably.  But, once we update the way we handle mdx, this might all be a moot point anyway.